src/rpc/request.rs

```diff
@@ -1023,18 +1021,34 @@ pub struct OfferInfo {
 }
 
 #[cfg_attr(feature = "serde", serde_as)]
-#[derive(Clone, PartialEq, Eq, Debug, Display, StrictEncode, StrictDecode)]
+#[derive(Clone, PartialEq, Eq, Debug, Display)]
 #[cfg_attr(
     feature = "serde",
     derive(Serialize, Deserialize),
     serde(crate = "serde_crate")
 )]
 #[display(TookOffer::to_yaml_string)]
 pub struct TookOffer {
-    pub offerid: PublicOfferId,
```

Comment:
Maybe, as mentioned in https://github.com/farcaster-project/farcaster-core/pull/292, we can use `[u8; 16]` to avoid manual impl. of encoding. (But with manual From/Into calls in code when we want `Uuid` types)

Reply:
TookOffer is printed as a yaml string, so I want to preserve the Uuid type. I am not sure how a `impl From` would help here.